### ISSUE TITLE
Cannot initialize library following example setup docs `NameError: uninitialized constant DingSDK::Ding::SERVER_PRODUCTION`

### ISSUE THREAD

**Issue by noahpryor** [ding-live/ding-ruby#1]
I'm getting an error that I believe is intrinsic to the gem (e.g. not a problem on my side) when trying to configure the library. 

Currently using ruby 3.1.4

Running ```DingSDK::Ding.new``` gives the following error:
```ruby
NameError: uninitialized constant DingSDK::Ding::SERVER_PRODUCTION

      server = SERVER_PRODUCTION if server.nil?
               ^^^^^^^^^^^^^^^^^
from //gems/ding_sdk-0.7.4/lib/ding_sdk/ding.rb:53:in `initialize'
```
Passing in a server argument `DingSDK::Ding.new(server: "beehiiv")`
Gets you
```ruby
TypeError: Parameter 'server_idx': Expected type T.nilable(Integer), got type String with value "beehiiv"
```
Trying passing in an integer gives the following error

```ruby
TypeError: Parameter 'server': Expected type String, got type Integer with value 1
```

Additionally, I'm pretty sure the syntax for the setup steps given during onboarding is not correct, though the above errors keep me from fully validating.
```ruby
s.config_security(
  security=Shared::Security.new(
    api_key="",
  )
)
```
`Shared::Security` needs to be namespaced under `DingSDK::Shared::Security`, and values should probably be assigned here as attributes of a hash rather than `security=` based on my quick skim of the code.

Finally, it's pretty aggressive to require sorbet as a runtime dependency here, but I can work around that if the above works.

**Comment by vbatoufflet** [comment]
Hi @noahpryor,

Thank you for your report!

There's indeed an issue with the SDK. While we'll be working on a proper resolution to this situation, here is a snippet to use as a temporary workaround:

```ruby
require 'ding_sdk'
require 'faraday'
require 'faraday/multipart'

client = Faraday.new(request: {
  params_encoder: Faraday::FlatParamsEncoder
}) do |f|
f.request :multipart, {}
end

configuration = ::DingSDK::SDKConfiguration.new(client, nil, "https://api.ding.live/v1",nil)
configuration.security = ::DingSDK::Shared::Security.new(
  api_key: "<YOUR_API_KEY>",
)

otp = ::DingSDK::Otp.new(configuration)

req = ::DingSDK::Shared::CreateAuthenticationRequest.new(
  customer_uuid: "<YOUR_CUSTOMER_UUID>",
  phone_number: "<PHONE_NUMBER>",
)

res = otp.create_autentication(req)
if ! res.create_authentication_response.nil?
  # handle response
end
```

**Comment by aym-v** [comment]
Issue is fixed in latest version (0.8.0). Thanks for reporting 👍🏻 

I'm moving the second point to a [new issue](https://github.com/ding-live/ding-ruby/issues/2)
